ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -292,14 +292,17 @@ Get Model Inference
     ...    set to ${TRUE}.
     [Arguments]    ${model_name}    ${inference_input}    ${token_auth}=${FALSE}   ${project_title}=${NONE}
     ...    ${kserve_mode}=Serverless   ${deployment_mode}=UI   ${service_port}=8888   ${end_point}=${NONE}
-    ...    ${service_account_name}=default-name   ${token}=${NONE}
+    ...    ${service_account_name}=default-name   ${token}=${NONE}      ${application_type}=${FALSE}
```

Comment:
`application_type` should be renamed to something more descriptive e.g `set_json_content_type` or alike

Reply:
ok let me change